com.unity.render-pipelines.high-definition/Documentation~/Environment-Lighting.md

```diff
@@ -44,7 +44,7 @@ The **Environment (HDRP)** section is at the top and has two settings that you c
 | ----------------------- | ------------------------------------------------------------ |
 | **Profile**             | A [Volume Profile](Volume-Profile.md) for the sky. This Volume Profile must include at least one Sky Volume override. |
 | **Static Lighting Sky** | The sky to use for the indirect ambient light simulation. The drop-down only contains sky types that the **Profile** includes. For example, if the **Profile** includes a **Gradient Sky** Volume override, you can select **Gradient Sky** from this drop-down.<br/>You can only edit this setting if you assign a Volume Profile to the **Profile** field. |
-| **Static Lighting Clouds** | The clouds to use for the indirect ambient light simulation. The drop-down only contains cloud types that the **Profile** includes.<br/>You can only edit this setting if you assign a Volume Profile to the **Profile** field. |
+| **Static Lighting Volumetric Clouds** | The clouds to use for the indirect ambient light simulation. The drop-down only contains cloud types that the **Profile** includes.<br/>You can only edit this setting if you assign a Volume Profile to the **Profile** field. The interaction between the Scene Volume sky type and the Scene's dominant directional light may affect the visual characteristics of this bake. |
```

Comment:
Ok, thank you.
Although this makes me realize---there doesn't seem to be a description of Volumetric Clouds in the doc, just Static Lighting Clouds. I've added Julien to this PR  as well because he did the original PR that prompted me to update the doc, I should have done that to start but forgot and GitHub suggested you.

So, although the screenshot is now up to date, I'm not sure where to add the extra information about the impact of the dominant directional light.